scripts/prox_crosswalk.py

```diff
@@ -34,7 +34,7 @@
 _YELP_ID_TO_PLACE_CACHE = {}
 
 def getAndCacheProviderIDs(keyID, providersList, identifiers):
-    providerIDs = _getCachedIDsForPlace(keyID, providersList)
+    providerIDs = {k: v for k,v in _getCachedIDsForPlace(keyID, providersList).items() if k in providersList}
```

Comment:
Oh good call, I think I added this while debugging the mysterious for loop bug, and forgot to take it out. Will undo.